library/docs/useStreetViewPanorama.md

```diff
@@ -0,0 +1,83 @@
+# `useStreetViewPanorama` Hook
+
+React hook to use the [Street View Panorama](https://developers.google.com/maps/documentation/javascript/streetview) in any component.
+
+## Usage
+The `StreetViewPanorama` can either be used within a DOM element, like a `<div>` element:
+
+```tsx
+import React, {useEffect} from 'react';
+import {
+  useGoogleMap,
+  useStreetViewPanorama
+} from '@ubilabs/google-maps-react-hooks';
+
+const MyComponent = () => {
+const [divContainer, setDivContainer] = useState<HTMLDivElement | null>(null);
+
+  const divRef = useCallback(
+    (node: React.SetStateAction<HTMLDivElement | null>) => {
+      node && setDivContainer(node);
+    },
+    []
+  );
+
+  const map = useGoogleMap();
+
+  const position = /** google.maps.LatLng */;
+  const pov = /** google.maps.StreetViewPov */;
+
+  const panorama = useStreetViewPanorama({
+    divElement: divContainer,
+    position,
+    pov
+  });
+
+  return <div className={styles.pano} ref={divRef} />;
+};
+```
+
+or be created on its own to be used by the map:
```

Comment:
Maybe add a note, that a map instance is only created, if the user passes a `mapContainer` to the `GoogleMapsProvider` and that this will be the map which is used here.

Reply:
Yes, very good idea 👍